src/main/webapp/resources/js/pages/projects/share/ShareProject.jsx

```diff
@@ -1,44 +1,28 @@
 import { Select, Space, Typography } from "antd";
 import React from "react";
 import { useDispatch, useSelector } from "react-redux";
-import {
-  useGetPotentialProjectsToShareToQuery
-} from "../../../apis/projects/projects";
+import { useGetPotentialProjectsToShareToQuery } from "../../../apis/projects/projects";
 import { setProject } from "./shareSlice";
 
 /**
  * React component for selecting the project to share a sample with.
+ * @param {list} projects - list of projects that the user is a manager on
  * @returns {JSX.Element}
  * @constructor
  */
-export function ShareProject() {
+export function ShareProject({ projects }) {
```

Comment:
Can we get the projects from the state here instead of passing it in?

Reply:
Ignoring :)